.bazelrc

```diff
@@ -1,3 +1,6 @@
+# Container Runtime
+build --action_env=CONTAINER_CMD=/usr/bin/docker
```

Comment:
why do we need to set this?

Reply:
In case both docker and podman are installed this is used to force the use of one of them. This could be useful to test builds with both container runtimes without having to remove podman to test docker.